pkg/actuators/machine/reconciler.go

```diff
@@ -18,24 +18,43 @@ package machine
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
+	"strings"
 	"time"
 
 	"github.com/IBM/vpc-go-sdk/vpcv1"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
 	machinev1 "github.com/openshift/api/machine/v1beta1"
 	machinecontroller "github.com/openshift/machine-api-operator/pkg/controller/machine"
 	"github.com/openshift/machine-api-operator/pkg/metrics"
-	ibmcloudproviderv1 "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
 	apicorev1 "k8s.io/api/core/v1"
 	apimachineryerrors "k8s.io/apimachinery/pkg/api/errors"
 	klog "k8s.io/klog/v2"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	ibmcloudclienterrors "github.com/openshift/machine-api-provider-ibmcloud/pkg/actuators/client/errors"
+	ibmcloudproviderv1 "github.com/openshift/machine-api-provider-ibmcloud/pkg/apis/ibmcloudprovider/v1"
 )
 
 const (
 	requeueAfterSeconds      = 20
 	requeueAfterFatalSeconds = 180
 	userDataSecretKey        = "userData"
+
+	// The following values are used for machine replacement, when it appears the machine is stuck and unresponsive as part of
+	// https://issues.redhat.com/browse/OCPBUGS-1327
+	// Time in minutes a Provisioned machine has before a delete is called to force re-create
+	machineReplaceDeadline = 15
+	machineDeleteDeadline  = 10
```

Comment:
i like the comment here, i might rename these variable to include the word `Minutes` in their name to make it completely explicit wherever they are used. eg
```suggestion
	machineReplaceDeadlineMinutes = 15
	machineDeleteDeadlineMinutes  = 10
```

Reply:
will do

Reply:
updated